bridge/slack/helpers.go

```diff
@@ -63,6 +63,18 @@ func (b *Bslack) getChannelBy(lookupKey string, lookupMap map[string]*slack.Chan
 
 const minimumRefreshInterval = 10 * time.Second
 
+func (b *Bslack) populateUser(userID string) {
+	user, err := b.sc.GetUserInfo(userID)
+	if err != nil {
```

Comment:
Only thing that I would add here is handling rate-limiting correctly.

You can see the typical way of doing this in the code of the `populateUsers()` method below.

Reply:
I decided to check the cache instead of implementing rate-limiting code.  This is much simpler.